packages/react/alert-dialog/src/AlertDialog.tsx

```diff
@@ -39,7 +39,7 @@ const [
 
 type AlertDialogContentPrimitive = Polymorphic.ForwardRefComponent<
   Polymorphic.IntrinsicElement<typeof DialogPrimitive.Content>,
-  Polymorphic.OwnProps<typeof DialogPrimitive.Content>
+  Omit<Polymorphic.OwnProps<typeof DialogPrimitive.Content>, 'onPointerDownOutside'>
```

Comment:
I think we normally lift this up when there are modifications, most of the time really.